tfhe/src/integer/ciphertext/compressed_ciphertext_list.rs

```diff
@@ -195,4 +197,173 @@ mod tests {
 
         assert!(cks.decrypt_bool(&decompressed3));
     }
+
+    const NB_TESTS: usize = 10;
+    const NB_OPERATOR_TESTS: usize = 1;
+    #[test]
+    fn test_ciphertext_compression() {
```

Comment:
Don't you want to remove the existing test to keep only this one?

Reply:
Indeed.